src/BuildingBlocks/EventBus/EventBusRabbitMQ/EventBusRabbitMQ.cs

```diff
@@ -240,20 +241,20 @@ private async Task ProcessEvent(string eventName, string message)
 
         if (_subsManager.HasSubscriptionsForEvent(eventName))
         {
-            await using var scope = _autofac.BeginLifetimeScope(AUTOFAC_SCOPE_NAME);
+            using var scope = _serviceProvider.CreateScope();
```

Comment:
To keep the semantics the same, we should use the `CreateAsyncScope` extension method instead (and keep the `await using`)